modules/samples/microprofile/microprofile-jwt/src/main/resources/configure_sample.sh

```diff
@@ -0,0 +1,523 @@
+#!/bin/sh
+
+# ----------------------------------------------------------------------------
+#  Copyright 2018 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+cleanup() {
+  sp_name=$1
+  delete_users
+  delete_roles
+  delete_sp "${sp_name}" urn:deleteApplication https://localhost:9443/services/IdentityApplicationManagementService.IdentityApplicationManagementServiceHttpsSoap11Endpoint/
+
+  return 0;
+}
+
+add_users_and_roles() {
+  IS_name=$1
+  IS_pass=$2
+  request_data1="configs/add-role-debtor.xml"
+  request_data2="configs/add-role-creditor.xml"
+  request_data3="configs/add-role-viewbalance.xml"
+
+  if [ ! -d "configs" ]
+    then
+      echo "configs Directory does not exists."
+      return 255
+  fi
+
+  if [ ! -f "$request_data1" ]
+    then
+      echo "$request_data1 File does not exists."
+      return 255
+  fi
+
+  if [ ! -f "$request_data2" ]
+   then
+      echo "$request_data2 File does not exists."
+      return 255
+  fi
+
+  if [ ! -f "$request_data3" ]
+    then
+      echo "$request_data2 File does not exists."
+      return 255
+  fi
+
+  echo
+  echo "Creating a user named cameron..."
+
+  # The following command can be used to create a user.
+  curl -s -k --user "${IS_name}":"${IS_pass}" --data '{"schemas":[],"name":{"familyName":"Smith","givenName":"Cameron"},"userName":"cameron","password":"cameron123","emails":"cameron@gmail.com","addresses":{"country":"United States"}}' --header "Content-Type:application/json" -o /dev/null https://localhost:9443/wso2/scim/Users
+  res=$?
+  if test "${res}" != "0"; then
+    echo "!! Problem occurred while creating user cameron. !!"
+    echo
+    return 255
+  fi
+  echo "** The user cameron was successfully created. **"
+  echo
+
+  echo "Creating a user named alex..."
+
+  curl -s -k --user "${IS_name}":"${IS_pass}" --data '{"schemas":[],"name":{"familyName":"Miller","givenName":"Alex"},"userName":"alex","password":"alex123","emails":"alex@gmail.com","addresses":{"country":"United States"}}' --header "Content-Type:application/json" -o /dev/null https://localhost:9443/wso2/scim/Users
```

Comment:
let's change the mail to some dummy one later..

Reply:
+1